parquet/src/test/java/org/apache/iceberg/parquet/TestParquetSchemaUtil.java

```diff
@@ -204,6 +204,54 @@ public void testSchemaConversionForHiveStyleLists() {
     Assert.assertEquals("Schema must match", expectedSchema.asStruct(), actualSchema.asStruct());
   }
 
+  @Test
+  public void testSchemaConversionForHiveStyleTwoLevelList() {
+    String messageType =
+            "message root {" +
+            "  required group f0 {" +
+            "    required group f00 (LIST) {" +
+            "      repeated group element {" +
+            "        required int32 f000;" +
+            "        optional int64 f001;" +
+            "      }" +
+            "    }" +
+            "  }" +
+            "}";
+
+    MessageType parquetScehma = MessageTypeParser.parseMessageType(messageType);
+    Schema expectedSchema = new Schema(
+            required(1, "f0", Types.StructType.of(
```

Comment:
Nit: A lot of this looks to be over-indented. Here, I would expect `required` to be indented 4 spaces after the start of `Schema` on the line above it.

Do you use IntelliJ? If so, there's a style guide checked in you can use. It doesn't handle everything, but it will handle the most common concerns (indentation, etc).

Instructions for Code formatter: https://iceberg.apache.org/#community/#configuring-code-formatter-for-intellij-idea

Reply:
Thanks @kbendick , I just did that and updated the PR as well.